hostprocess/calico/kube-calico-rbac.yml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: v1
```

Comment:
should these roles already be present from the Linux calico deployment?

Reply:
From what I tested, they weren't and needed to be applied separately

Reply:
ah, I think this might be due to the way calico is installed here. https://projectcalico.docs.tigera.io/manifests/tigera-operator.yaml vs https://github.com/projectcalico/calico/releases/download/$(CALICO_VERSION)/release-$(CALICO_VERSION).tgz 

I used the released files when developing the files in the calico folder. I bet if those are used we wouln't need this. 

I don't really know the difference between the release files and operator. do you?

Reply:
Unfortunatelly, I don't know either

Reply:
Given the object naming is different, I think it makes sense to use the files in to be consistent  https://github.com/projectcalico/calico/releases/download/$(CALICO_VERSION)/release-$(CALICO_VERSION).tgz. Otherwise we end up with two roles that are basically the same for just named different because of the operator. 

btw, thanks for all the hard work on this. I am testing the changes locally but things look lgtm

Reply:
Updated the guide to use calico from projectcalico! If there is something that I might have missed or can be modified, please let me know!